src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1487,8 +1483,9 @@ def enable_features(cmd, client, resource_group_name, cluster_name, features, ku
         cmd_helm_upgrade.extend(["--kube-context", kube_context])
     if enable_azure_rbac:
         cmd_helm_upgrade.extend(["--set", "systemDefaultValues.guard.enabled=true"])
-        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.guard.clientId={}".format(azrbac_client_id)])
-        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.guard.clientSecret={}".format(azrbac_client_secret)])
+        # Setting the default authnMode mode as "arc" for guard. This mode uses PoP token based auth. and Arc RBAC 1P apps for authN/authZ.
+        cmd_helm_upgrade.extend(["--set", "systemDefaultValues.guard.authnMode=arc"])
+        logger.warning("Please use the kubelogin version v0.0.32 or higher which has support for generating PoP token(s) which is needed by guard running in 'arc' authN mode.")
```

Comment:
Nit: maybe reword this as:
"Please use kubelogin version v0.0.32 or higher, which has support for generating PoP token(s). This is required by guard when running in 'arc' authN mode."

Reply:
If you change the default behavior, you need to confirm the impact of the breaking change on the customers. 
We usually recommend implementing the breaking change in the breaking change window (**Ignite** and **Build** sprint), and before that, we suggest recommending customers to change their behavior through notification messages (warning log)